pkg/skaffold/server/server.go

```diff
@@ -196,9 +207,21 @@ func newGRPCServer(preferredPort int, usedPorts *util.PortSet) (func() error, in
 }
 
 func newHTTPServer(preferredPort, proxyPort int, usedPorts *util.PortSet) (func() error, error) {
-	mux := runtime.NewServeMux(runtime.WithProtoErrorHandler(errorHandler), runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.JSONPb{OrigName: true, EmitDefaults: true}))
```

Comment:
What is the reason the errorHandler is removed?

Reply:
In grpc-gateway v2, Error handling configuration has been overhauled.
_runtime.HTTPError, runtime.OtherErrorHandler, runtime.GlobalHTTPErrorHandler, runtime.WithProtoErrorHandler are all gone. Error handling is rewritten around the use of gRPCs Status types._
Custom Error handling function may not be required anymore.

Reply:
I wonder how we can test this